docs/upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -7,6 +7,13 @@ There you can find links to upgrade notes for other versions too.
 
 ## [shopsys/framework]
 
+### Infrastructure
+- update your `kubernetes/deployments/webserver-php-fpm.yml` file: ([#1368](https://github.com/shopsys/shopsys/pull/1368))
+    ```diff
+    -   command: ["sh", "-c", "cd /var/www/html && ./phing db-create dirs-create db-demo product-search-recreate-structure product-search-export-products grunt error-pages-generate warmup"]
+    +   command: ["sh", "-c", "cd /var/www/html && ./phing -D productionConfirmDestructiveAction=y db-create dirs-create db-demo product-search-recreate-structure product-search-export-products grunt error-pages-generate warmup"]
```

Comment:
there should be `production.confirm.destructive.action` (and `production.confirm.action` after 6845dcb204b04115e0fdc94be8ad4c4c8f4c3940)